cpp/src/io/parquet/page_data.cu

```diff
@@ -862,19 +862,17 @@ static __device__ void gpuOutputGeneric(volatile page_state_s* s,
 /**
  * @brief Sets up block-local page state information from the global pages.
  *
- * @param[in, out] s The local page state to be filled in
- * @param[in] p The global page to be copied from
- * @param[in] chunks The global list of chunks
- * @param[in] num_rows Maximum number of rows to read
- * @param[in] min_row crop all rows below min_row
- * @param[in] num_chunk Number of column chunks
+ * @param s The local page state to be filled in
```

Comment:
Is the page state is still an output parameter?

Reply:
Yes.

Reply:
If so, we should probably keep the `in/out` indicators since not all parameters are input.

Reply:
I removed them because that's been the new standard in cudf for a while.  I can put them back but in general I see this kind of stuff just being removed.   @vuule

Reply:
Done.